pom.xml

```diff
@@ -85,6 +85,15 @@
     <build>
         <plugins>
             <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-compiler-plugin</artifactId>
+                <version>3.2</version>
+                <configuration>
+                    <source>1.7</source>
+                    <target>1.7</target>
```

Comment:
To get a reproducable build, we should really set the source and target version. That should not prevent Travis CI compiling it with JDK8, I think. Compiling it with JDK6 won't work anyway, if the language level is set to 1.7, which allows us to use Java 7 features.